packages/itwinui-css/src/radio-tile/radio-tile.scss

```diff
@@ -124,7 +124,8 @@
   }
 
   @include focus-visible {
-    + * { // outline for checkmark
+    + * {
+      // outline for checkmark
```

Comment:
This looks unintentional, lets undo

Reply:
Yikes, I don't know how that happened!  I've reverted it but when trying to commit am getting an error:
```
> git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file -
husky > pre-commit (node v16.14.0)
⚠ Detected incorrect braces with only single value: `*.{scss}`. Reformatted as: `*.scss`

[STARTED] Preparing...
[SUCCESS] Preparing...
[STARTED] Running tasks...
[STARTED] Running tasks for *.scss
[STARTED] Running tasks for *.{scss,html}
[STARTED] Running tasks for *.{scss,html,js}
[STARTED] prettier --write
[STARTED] stylelint --fix
[STARTED] yarn lint:copyright --fix
[SUCCESS] prettier --write
[SUCCESS] Running tasks for *.scss
[SUCCESS] yarn lint:copyright --fix
[SUCCESS] Running tasks for *.{scss,html,js}
[SUCCESS] stylelint --fix
[SUCCESS] Running tasks for *.{scss,html}
[SUCCESS] Running tasks...
[STARTED] Applying modifications...
[FAILED] Prevented an empty git commit!
[STARTED] Reverting to original state because of errors...
[SUCCESS] Reverting to original state because of errors...
[STARTED] Cleaning up...
[SUCCESS] Cleaning up...

  ⚠ lint-staged prevented an empty git commit.
  Use the --allow-empty option to continue, or check your task configuration

husky > pre-commit hook failed (add --no-verify to bypass)
```

I used `git commit --no-verify` to get it to commit.

Reply:
I'm seeing a lot of unintended changes in other components again, can you take a look?